src/main/webapp/i18n/en/exam.json

```
@@ -453,6 +454,83 @@
                 }
             }
         },
+        "examMonitoring": {
+            "title": "Exam Monitoring",
+            "description": "The Exam Monitoring Dashboard allows instructors to monitor students' actions during the exam and analyze the resulting data.",
+            "table": {
+                "panelHeader": "Exam Information",
+                "panelDescriptionHeader": "Details",
+                "title": "Title",
+                "start": "Start",
+                "end": "End",
+                "students": "Registered students",
+                "exercises": "Exercises",
+                "exerciseGroups": "Exercise groups"
+            },
+            "menu": {
+                "overview": {
+                    "title": "Overview"
+                },
+                "exercises": {
+                    "title": "Exercises",
+                    "description": "Shows the current participation of the students in the respective exercise and exercise groups",
+                    "groups": "Exercise Groups"
+                },
+                "activity-log": {
+                    "title": "Activity log",
+                    "description": "Shows the past and current activities of active students"
+                }
+            },
+            "charts": {
+                "exercise-chart": {
+                    "name": "Current amount of students per exercise grouped by exercise group",
+                    "tooltip": "Shows the current amount of students per exercise grouped by exercise group",
+                    "xAxisLabel": "Amount of students",
```

```suggestion
                    "xAxisLabel": "Number of students",
```
The group of students has a countable number of members.
The same applies to the other translations for amount of students/submissions/actions/… as well.